latex.ddoc

```diff
@@ -3,6 +3,7 @@ _=Macros for LaTeX generation. The following section is ordered alphabeticaly.
 _=*****************************************************************************
 
 A=\ref{$1}{$+}
+AHTTP=\href{http://$1}{$+}
```

Comment:
I have no idea. There's also `HTTP`. The differences (if any) between them are not clear to me at all.